<ui version="4.0" >
 <class>KProxyDialogUI</class>
 <widget class="QWidget" name="KProxyDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>473</height>
   </rect>
  </property>
  <property name="whatsThis" >
   <string>&lt;qt>
Setup proxy configuration.
&lt;p>
A proxy server is an intermediate machine that sits between your computer and the Internet and provides services such as web page caching and filtering. Caching proxy servers give you faster access to web sites you have already visited by locally storing or caching those pages; filtering proxy servers usually provide the ability to block out requests for ads, spam, or anything else you want to block.
&lt;p>
If you are uncertain whether or not you need to use a proxy server to connect to the Internet, consult your Internet service provider's setup guide or your system administrator.
&lt;/qt></string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="Q3ButtonGroup" name="gbConfigure" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="exclusive" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="rbNoProxy" >
        <property name="whatsThis" >
         <string>Connect to the Internet directly.</string>
        </property>
        <property name="text" >
         <string>Connect to the &amp;Internet directly</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAutoDiscover" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="whatsThis" >
         <string>&lt;qt>
Automatically detect and configure the proxy settings.&lt;p>
Automatic detection is performed using the &lt;b>Web Proxy Auto-Discovery Protocol (WPAD)&lt;/b>.&lt;p>
&lt;b>NOTE:&lt;/b> This option might not work properly or not work at all in some UNIX/Linux distributions. If you encounter a problem when using this option, please check the FAQ section at http://konqueror.kde.org.
&lt;/qt></string>
        </property>
        <property name="text" >
         <string>A&amp;utomatically detect proxy configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAutoScript" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="whatsThis" >
         <string>Use the specified proxy script URL to configure the proxy settings.</string>
        </property>
        <property name="text" >
         <string>U&amp;se the following proxy configuration URL</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KUrlRequester" name="location" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis" >
           <string>Enter the address for the proxy configuration script.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rbEnvVar" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis" >
           <string>&lt;qt>
Use environment variables to configure the proxy settings.&lt;p>
Environment variables such as &lt;b>HTTP_PROXY&lt;/b> and &lt;b>NO_PROXY&lt;/b> are usually used in multi-user UNIX installations, where both graphical and non-graphical applications need to share the same proxy configuration information.
&lt;/qt></string>
          </property>
          <property name="text" >
           <string>Use preset proxy environment &amp;variables</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbEnvSetup" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="whatsThis" >
           <string>Show the proxy environment variable configuration dialog.</string>
          </property>
          <property name="text" >
           <string>Setup...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rbManual" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis" >
           <string>Manually enter proxy server configuration information.</string>
          </property>
          <property name="text" >
           <string>&amp;Manually specify the proxy settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbManSetup" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="whatsThis" >
           <string>Show the manual proxy configuration dialog.</string>
          </property>
          <property name="text" >
           <string>Setup...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3ButtonGroup" name="gbAuth" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Authori&amp;zation</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="rbPrompt" >
        <property name="whatsThis" >
         <string>Prompt for login information whenever it is required.</string>
        </property>
        <property name="text" >
         <string>Prompt as &amp;needed</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPresetLogin" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="whatsThis" >
         <string>Use the information below to login into proxy servers as needed.</string>
        </property>
        <property name="text" >
         <string>Use the following lo&amp;gin information.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item rowspan="2" row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2" >
         <widget class="KLineEdit" name="lePassword" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>Login password.</string>
          </property>
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="KLineEdit" name="leUsername" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>Login name.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="lbPassword" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>Login password.</string>
          </property>
          <property name="text" >
           <string>Password:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="lbUsername" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>Login name.</string>
          </property>
          <property name="text" >
           <string>Username:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3ButtonGroup" name="gbOptions" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>O&amp;ptions</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="cbPersConn" >
        <property name="whatsThis" >
         <string>&lt;qt>
Use persistent proxy connection.&lt;p>
Although a persistent proxy connection is faster, note that it only works correctly with proxies that are fully HTTP 1.1 compliant. Do &lt;b>not&lt;/b> use this option in combination with non-HTTP 1.1 compliant proxy servers such as JunkBuster and WWWOfle.
&lt;/qt></string>
        </property>
        <property name="text" >
         <string>Use persistent connections to proxy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>369</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>rbPresetLogin</sender>
   <signal>toggled(bool)</signal>
   <receiver>leUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbPresetLogin</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbPresetLogin</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbPresetLogin</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbAutoScript</sender>
   <signal>toggled(bool)</signal>
   <receiver>location</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
